CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+# Upcoming Release
```

Comment:
I added this as a place to make note of version changes and upcoming changes. Does this sound like a good plan? We could then copy over the notes from here when tagging the release.